src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -854,16 +854,10 @@ void AppCommandlineArgs::FullResetState()
     _exitMessage = "";
     _shouldExitEarly = false;
 
-    _windowTarget = std::nullopt;
+    _windowTarget = "";

```

Comment:
```suggestion
    _windowTarget = {};
```

this one is more nitty than the below